insonmnia/miner/gpu/nvidia_tuner.go

```diff
@@ -58,26 +143,37 @@ func newNvidiaTuner(ctx context.Context, opts ...Option) (Tuner, error) {
 
 	defer func() { nvidia.Shutdown() }()
 
-	log.G(ctx).Info("NVIDIA GPU supported by the host. Discovering GPU devices")
```

Comment:
restore

Reply:
better to prefer comma instead of dot

Reply:
[Removed!](https://github.com/sonm-io/core/pull/491/commits/6688596f41d7ba69e1c67a40862d0c5a09321315?diff=split#diff-f6057e6d60f924533a29826beee17231R146)